src/main/java/dk/kb/present/api/v1/impl/ServiceApiServiceImpl.java

```diff
@@ -72,6 +75,8 @@ public StatusDto status() throws ServiceException {
                 .java(System.getProperty("java.version"))
                 .heap(Runtime.getRuntime().maxMemory()/1048576L)
                 .server(host)
+                .gitCommitChecksum(BuildInfoManager.getGitCommitChecksum())
+                .gitBranch(BuildInfoManager.getGitBranch())
```

Comment:
What about the tags and commit time?

Reply:
Should be added as well. I think I've been going to much forward between this one and the `kb-util` part. I'll add it.